### ISSUE TITLE
Keeper 0.6.0 fails in barge.

### ISSUE THREAD

**Issue by ssallam** [oceanprotocol/keeper-contracts#273]
Running the keeper-contarcts v0.6.0 in barge start_ocean.sh failes with this error:
```
keeper-contracts_1           | > @oceanprotocol/keeper-contracts@0.6.0 migrate /keeper-contracts
keeper-contracts_1           | > ./scripts/deployContracts.sh "--network=spree"
keeper-contracts_1           | 
keeper-contracts_1           | + stake=10
keeper-contracts_1           | + maxSlots=1
keeper-contracts_1           | + NETWORK=development
keeper-contracts_1           | ++ jq -r .version package.json
keeper-contracts_1           | ./scripts/deployContracts.sh: line 14: jq: command not found
keeper-contracts_1           | + VERSION=v
keeper-contracts_1           | npm ERR! file sh
keeper-contracts_1           | npm ERR! code ELIFECYCLE
keeper-contracts_1           | npm ERR! errno ENOENT
keeper-contracts_1           | npm ERR! syscall spawn
keeper-contracts_1           | npm ERR! @oceanprotocol/keeper-contracts@0.6.0 migrate: `./scripts/deployContracts.sh "--network=spree"`
keeper-contracts_1           | npm ERR! spawn ENOENT
keeper-contracts_1           | npm ERR! 
keeper-contracts_1           | npm ERR! Failed at the @oceanprotocol/keeper-contracts@0.6.0 migrate script.
keeper-contracts_1           | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
keeper-contracts_1           | 
keeper-contracts_1           | npm ERR! A complete log of this run can be found in:
keeper-contracts_1           | npm ERR!     /root/.npm/_logs/2019-01-21T15_49_59_085Z-debug.log
```
Sebastian mentioned that the jq package is missing in the docker build.

**Comment by H34D** [comment]
related to #266

**Comment by H34D** [comment]
https://github.com/oceanprotocol/keeper-contracts/pull/220/commits/3f45be6f63104e625952657eba23ca47860d4f34

**Comment by H34D** [comment]
closes with #220
